.github/workflows/release_pipeline.yml

```
@@ -0,0 +1,320 @@
+name: Release Pipeline
+
+# Triggers automatically on push to main when any version file changes
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)